library/alloc/src/slice.rs

```diff
@@ -220,6 +220,7 @@ mod hack {
 }
 
 #[lang = "slice_alloc"]
+#[cfg_attr(not(test), rustc_diagnostic_item = "slice")]
```

Comment:
I'm not sure on this being `slice` -- I see the symbol was already there, but as far as I can tell wasn't being used anywhere.

Reply:
Seems good to me. `sym::slice` is used in `rustc_hir::lang_items::LangItem::Slice::name()` but it doesn't really matter.